service/ziti-tunnel/service/ipc.go

```diff
@@ -1142,11 +1142,13 @@ func handleBulkServiceChange(sc cziti.BulkServiceChange) {
 	rts.BroadcastEvent(be)
 
 	id := rts.Find(sc.Fingerprint)
-	var m = dto.IdentityEvent{
-		ActionEvent: dto.IdentityUpdateComplete,
-		Id: Clean(id),
+	if id != nil {
```

Comment:
shouldn't this be done from wherever `handleBulkServiceChange` is called instead? this will still end up sending a bulkserviceevent - is that on purpose?